SRT/CDB/alma/Procedures/StationProcedures/StationProcedures.xml

```diff
@@ -20,11 +20,18 @@
 <Procedure name="setupCCG" args="0">
 <body>
 	receiversSetup=CCG
+	antennaSetup=CCG
 	chooseBackend=TotalPower
+	initialize=PPP
```

Comment:
@sergio the "PPP" configuration of the focus selector sets the 730MHz filter cutting the receiver band. If this is not intended I suggest to add a CCG specific configuration

Reply:
@aorlati I do agree. Don't know how to do it.

Reply:
With setSection it's possibile to setup the choosen band also with PPP

Reply:
Yes it's possible or you replace `initialize=CCG` and you add a 'CCG' configuration into this file: https://github.com/discos/discos/blob/master/SRT/Configuration/CDB/alma/DataBlock/TotalPower/TotalPower.xml
You can copy the PPP, change the configurationID and inputPortBw.